app/js/lib/foxbox/api.js

```diff
@@ -224,6 +230,28 @@ export default class API {
   }
 
   /**
+   * Returns promise that is resolved once document becomes visible.
+   *
+   * @returns {Promise}
+   * @private
+   */
+  [p.onceDocumentVisible]() {
```

Comment:
It seems that there is no need in doing any requests when document is not active.